cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -3152,6 +3242,21 @@
                     "cfn_efs_shared_dir": {
                       "Ref": "EFSOptions"
                     },
+                    "cfn_fsx": {
+                      "Fn::If": [
+                        "CreateFSXSubstack",
+                        {
+                          "Fn::GetAtt": [
+                            "FSXSubstack",
+                            "Outputs.FileSystemId"
+                          ]
+                        },
+                        ""
+                      ]
+                    },
+                    "cfn_fsx_shared_dir": {
```

Comment:
`fsx_shared_dir`? `shared_dir` makes it consistent with the ebs section. For example:

```
[fsx fs]
shared_dir = /fsx

[ebs fs]
shared_dir = /shared
```

Reply:
I was referring to the fact that cfn_fsx_shared_dir which is used in user data contains the tuple with the entire FSx config and not only the shared_dir so it could be named something like `cfn_fsx_options`. I agree that the cluster config should remain as it is.

Reply:
changed `cfn_fsx_shared_dir -> cfn_fsx_options`, I also noticed that `cfn_fsx` should really be `cfn_fsx_fs_id` so I've changed that as well.